index.html

```diff
@@ -1,12 +1,801 @@
+                <div class="hero__flex-row">
+                  <div class="hero__icon">
+                    <div class="hero__carbon">
+                      <img
+                        class="hero__carbon-icon"
+                        src="images/icon3.JPG"
+                        alt=""
+                      />
+                    </div>
+                  </div>
+                  <div class="hero__carbon-heading">
+                    <h5 class="hero__heading-five">
+                      National top 50 firms
+                    </h5>
+                  </div>
+                </div>
+                <p class="hero__paragraph-two">
+                  The quick fox jumps over the lazy dog the quick fox jumps over
+                  the lazy dog
+                </p>
+              </div>
+            </div>
+            <div class="hero__col-four">
+              <div class="hero__card-two">
+                <div class="hero__flex-row">
+                  <div class="hero__icon">
+                    <div class="hero__carbon-down">
+                      <img
+                        class="hero__second-icon"
+                        src="images/icon4.JPG"
+                        alt=""
+                      />
+                    </div>
+                  </div>
+                  <div class="hero__carbon-heading">
+                    <h5 class="hero__heading-five">
+                      National top 50 firms
+                    </h5>
+                  </div>
+                </div>
+                <p class="hero__paragraph-two">
+                  The quick fox jumps over the lazy dog the quick fox jumps over
+                  the lazy dog
+                </p>
+              </div>
+            </div>
+            <div class="hero__col-four">
+              <div class="hero__card">
+                <div class="hero__flex-row">
+                  <div class="hero__icon">
+                    <div class="hero__carbon-down">
+                      <img
+                        class="hero__second-icon"
+                        src="images/icon5.JPG"
+                        alt=""
+                      />
+                    </div>
+                  </div>
+                  <div class="hero__carbon-heading">
+                    <h5 class="hero__heading-five">Web Development</h5>
+                  </div>
+                </div>
+                <p class="hero__paragraph-two">
+                  The quick fox jumps over the lazy dog the quick fox jumps over
+                  the lazy dog
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="clients">
+      <div class="clients__container">
+        <div class="clients__row">
+          <div class="clients__main-content">
+            <h2 class="clients__h2">We are providing best busines service.</h2>
+            <p class="clientients__paragraph">
```

Comment:
You have a typo here for `clientients__paragraph`, I think you meant `clients__paragraph`